parseRecipes.js

```diff
@@ -58,5 +62,19 @@ module.exports = function(recipeSlugs) {
       demo_index: recipe + '/index.html',
       intro_ref: recipe + '.html',
     };
+  }).sort(function(recipeA, recipeB) {
+    if (recipeA.category === recipeB.category) {
+      return recipeA.difficulty < recipeB.difficulty ? -1 : 1;
+    }
+
+    if (recipeA.category < recipeB.category) {
+      return -1;
+    }
+
+    return 1;
+  }).sort(function(recipeA, recipeB) {
+    // Sort recipes by preferred category
+    assert(categoryOrder.indexOf(recipeA.category) !== -1, recipeA.name + ': Unexpected category value "' + recipeA.category + '"');
```

Comment:
Ohh, your concern is the assert and not the correct sort?  OK, that makes sense.  Will update.